src/Linux/Host/Broker/Impl/Services/LinuxSystemInfoService.cs

```
@@ -111,8 +118,103 @@ public class LinuxSystemInfoService : ISystemInfoService {
+                                    var match = pattern.Match(dmesgData[i]);
+                                    if (match.Success) {
+                                        vram = GetRamValueMB(match.Value);
+                                        break;
+                                    }
+                                }
+                            }
+                        }
+                    }
+
+                    _videoCardInfo.Add(new VideoCardInfo() { VideoCardName = productName, VideoRAM = vram, VideoProcessor = vendorName });
+                    ++count;
+                }
+            }
+        }
+
+        private static long GetRamValueMB(string vramStr) {
+            long ram = 0;
+            Regex numPattern = new Regex("[0-9]+");
+            var match = numPattern.Match(vramStr);
+            
+            if (match.Success) {
+                long.TryParse(match.Value, out ram);
+            }
+
+            if (vramStr.ContainsIgnoreCase("G")) {
+                ram *= 1024;
+            }
+
+            return ram;
+        }
+
+        private IEnumerable<string> ExecuteAndGetOutput(string command, string arguments) {
+            Process proc = null;
+            List<string> standardOutData = new List<string>();
+            try {
+                ProcessStartInfo psi = new ProcessStartInfo();
+                psi.Arguments = arguments;
+                psi.FileName = command;
+                psi.RedirectStandardInput = true;
+                psi.RedirectStandardOutput = true;
+                psi.RedirectStandardError = true;
+                proc = _ps.Start(psi);
+
+                using (StreamReader reader = new StreamReader(proc.StandardOutput.BaseStream)) {
+                    while (!reader.EndOfStream) {
+                        standardOutData.Add(reader.ReadLine());
+                    }
+                }
+            } catch (Exception) {
```

Why do we swallow all errors?